oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -331,8 +332,19 @@ private APIGatewayProxyResponseEvent redirect(
                         });
 
         if (configurationService.isAuthOrchSplitEnabled()) {
-            var encryptedJWT =
-                    authorizationService.getSignedAndEncryptedJWT(client.getClientName());
+            var claims =
+                    new JWTClaimsSet.Builder()
+                            .claim("client-name", client.getClientName())
+                            .claim("cookie-consent-shared", client.isCookieConsentShared())
+                            .claim("consent-required", client.isConsentRequired())
+                            .claim("is-one-login-service", client.isOneLoginService())
+                            .claim("service-type", client.getServiceType())
+                            .claim("state", authenticationRequest.getState())
+                            .claim("scopes", authenticationRequest.getScope().toString())
+                            .claim("redirect-uri", authenticationRequest.getRedirectionURI())
```

Comment:
This will need to be the redirect URI of Orchestration rather than the RP redirect URI